packages/itwinui-react/src/core/Table/TablePaginator.tsx

```diff
@@ -175,9 +175,13 @@ export const TablePaginator = (props: TablePaginatorProps) => {
     (index: number, tabIndex = index === focusedIndex ? 0 : -1) => (
       <ButtonBase
         key={index}
-        className={cx('iui-table-paginator-page-button', {
-          'iui-table-paginator-page-button-small': buttonSize === 'small',
-        })}
+        className={cx(
+          'iui-button',
```

Comment:
This one should use `<Button>` directly. The `iui-button` class should not be used outside of `Button`.

Reply:
Ohh, yes, changed to `Button` (https://github.com/iTwin/iTwinUI/pull/1919/commits/be3b0d361fc694bb867fa9dda17a85328176bba7).